frontend/web/components/pages/OrganisationSettingsPage.js

```diff
@@ -228,6 +231,9 @@ const OrganisationSettingsPage = class extends Component {
                   const { chargebee_email } = subscriptionMeta || {}
 
                   const displayedTabs = []
+                  const isEnterprise =
+                    global.flagsmithVersion?.backend.is_enterprise
+                  const isSaas = global.flagsmithVersion?.backend?.isSaas
```

Comment:
I wonder if these should be moved to `utils.tsx` ? Are there other occurrences in the code base where we're checking these flags?

Reply:
Makes sense, we use that flag inside of Utils as well.